### PR TITLE
[CMPN-17] Add options to Radio PB Kit

### PR DESCRIPTION
**Pull request by ljtinney**
#### Screens

Screen Shot 2020-09-11 at 2 39 17 PM

Screen Shot 2020-09-11 at 2 39 08 PM

#### Breaking Changes

No. 
This PR adds the options prop to the radio kit so PlayBook users can pass every single kind of HTML element option to the input element. 
This PR does not remove any props as to not break any existing usages of the radio kit.

#### Runway Ticket URL

https://nitro.powerhrg.com/runway/backlog_items/CMPN-17

#### Checklist:

- [x] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `depreciation`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [x] **DEPLOY** Please add the `Milano` label when you are ready for a review.
- [x] **SCREENSHOT** Please add a screen shot or two.
- [ ] **SPECS** Please cover your changes with specs.

### CODE REVIEW

File: app/pb_kits/playbook/pb_date/docs/_date_timezone.html.erb [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<%= pb_rails("caption", props: { text: "East Coast (Default)"}) %>
```
Comment (darrendc) [inline app/pb_kits/playbook/pb_date/docs/_date_timezone.html.erb:1]:
```suggestion
<%= pb_rails("caption", props: { text: "East Coast (Default)" }) %>
```
Minor nitpick...you blank spaced after the left  `{` but not after the right. This goes for all uses of the caption and body kits throughout the rest of this file.

Comment (ljtinney) [reply app/pb_kits/playbook/pb_date/docs/_date_timezone.html.erb:1]: I shall add those extra blank spaces. (Taylor Swift would approve.)

**Comment by garettarrowood** [comment]
@ljtinney - Looks like you merged the wrong base branch into this PR and have generated a lot of diff.

At this point, it would likely be better to cherry-pick over your commits to a new PR. Be *sure* that the new branch is branched off of the right base branch. At the time of writing this, that is `6.7.0`.

**Comment by garettarrowood** [comment]
Closed in preference to #1099

**Pull request closed without merging**